[ASM] Expander NGFWv1 (#29831)
demisto/content: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_On_Prem_Enrichment_README.md

> Given and IP address, port and protocol of a service, this playbook enriches on-prem integrations to find the related firewall rule and other related information.

Reviewer: ```suggestion
Given an IP address, port, and protocol of a service, this playbook enriches on-prem integrations to find the related firewall rule and other related information.
```

Conditions:

> - This is currently limited to stand-alone firewalls for PAN-OS.

Reviewer: ```suggestion
This is currently limited to standalone firewalls for PAN-OS.
```

## Dependencies

This playbook uses the following sub-playbooks, integrations, and scripts.

### Sub-playbooks

This playbook does not use any sub-playbooks.

### Integrations

> * Panorama

Reviewer: ```suggestion
Panorama
```

Author: this was configured by SDK
